test1.py

```diff
@@ -0,0 +1,18 @@
+available = ["TOYOTA", "BMW", "MERCEDES", "HYUNDAI"]
+requested = ["FORD", "BMW", "MERCEDES"]
+offer = []
+index = 0
+for a in range(len(available)):
+	car = available[a]
+	for r in range(len(requested)):
+		car_two = requested[r]
+		if car == car_two:
+			offer.append(car)
+	index += 1
+re = "The following requested cars are available: "
+index_two = 0
+while(index_two < len(offer)):
```

Comment:
Instead of using `while` and `len` to iterate through the lists you can use `join`.

```
>>> available_offers 
['BMW', 'MERCEDES']
>>> "The following requested cars are available: " + ", ".join(available_offers) + "."
'The following requested cars are available: BMW, MERCEDES.'
```